jobs/assets-subsystem/asset-sender/src/main/java/com/paladincloud/assetsender/Main.java

```diff
@@ -0,0 +1,11 @@
+package com.paladincloud.assetsender;
+
+import com.paladincloud.common.DaggerServerComponent;
```

Comment:
Lets break apart from common concept. All new services/workers need to be isolated to its subsystem's domain, hence if you want to introduce common, it should be common only for that subsystem.